docs/tutorials/FiveMinuteIntro.md

```diff
@@ -20,15 +20,15 @@ At the end of this five minute introduction you will have:
 
 We host our Cromwell releases on GitHub! You can find the latest version on our [Releases](https://github.com/broadinstitute/cromwell/releases/latest) page.
 
-* Head down to the bottom of the page and find the link to download `cromwell-29.jar`. Note that when you read these instructions, the latest Cromwell version might be higher than 29. That's fine, these instructions will work just as well! Download the jar file.
- 	 * WARNING! If you're on a Mac, the security settings might try to stop you from running Cromwell! Don't worry, if this happens just go to `System Preferences > Security & Privacy > General` and find the `cromwell` jar listed on the page. Click `Open anyway`. The `cromwell-29.jar` will now automatically download to your `Downloads` directory.
+* Look for the latest version at the top of the page, and find the link to download the jar. It'll have a name like `cromwell-XYZ.jar` where `XYZ` is the version. Download the jar file.
+* WARNING! If you're on a Mac, the security settings might try to stop you from running Cromwell! Don't worry, if this happens just go to `System Preferences > Security & Privacy > General` and find the `cromwell` jar listed on the page. Click `Open anyway`. The `cromwell-XYZ.jar` will now automatically download to your `Downloads` directory.
```

Comment:
Maybe just `cromwell-xy.jar`? At least until we reach version 100?

Reply:
@cjllanwarne Your ability to out-pedantic me never ceases to am[aze|use] me

Reply:
I'll chose to take that as a complement